qiskit_ibm_runtime/options/twirling_options.py

```diff
@@ -33,7 +33,7 @@ class TwirlingOptions:
         enable_gates: Whether to apply 2-qubit gate twirling. Default: False.
 
         enable_measure: Whether to enable twirling of expectation value measurements
-          in Estimator. Default: True.
+          in the Primitive. Default: True.
```

Comment:
The wording should just say measurements, and the sentence on conditional should also be here, e.g. 

`
enable_measure: Whether to enable twirling of measurements. The twirling can only be applied to those measurement registers not involved within a conditional logic. Default: True.
`